README.md

```diff
@@ -38,34 +38,43 @@ Jina documentation adheres to the following hierarchical structure. Each Jina pr
 
 If you find a gap in our documentation, please submit a GitHub issue [here](https://github.com/jina-ai/docs/issues/new).
 
-## How to add pages
+## Documentation Process
```

Comment:
Nit: This is in title case while everything else is in sentence case.

Reply:
True, typo